svgdigitizer/electrochemistry/cv.py

```diff
@@ -23,20 +51,157 @@
 from functools import cache
 import re
 import matplotlib.pyplot as plt
-from pathlib import Path
 from astropy import units as u
 
 
 class CV():
-    # TODO: Add documentation with a usage example. #60
-    # Until documentation is added, this class will not show in the auto-generated documentation.
+    r"""
+    A digitized cyclic voltammogram (CV) derived from an SVG file, 
+    which provides access to the objects of the CV.
+
+    Typically, the SVG input has been created by tracing a CV from
+    a publication with a `<path>` in an SVG editor such as Inkscape. Such a
+    path can then be analyzed by this class to produce the coordinates
+    corrsponding to the original measured values.
+
+    TODO:: Link to workflow.md (see issue #73)
+
+    EXAMPLES:
+
+    An instance of this class can be created from a specially prepared SVG file.
+    It requires:
+
+    * that the label of the point x2 on the x-axis contains a value and a unit such as ``<text>x2: 1 mV</text>``.  Optionally, this text also contains a reference such as ``<text>x2: 1 mV vs. RHE</text>``.
+    * that the label of the point x2 on the y-axis contains a value and a unit such as ``<text>y2: 1 uA / cm2</text>``.
+    * that a rate is provided in a text field such as ``<text">scan rate: 50 V / s</text>`` or ``<text>rate: 50 mV / s</text>`` placed anywhere in the SVG file.
+
+    The data in the CV can be returned as a dataframe with axis 't', 'U', and 'I' (current) or 'j' (current density).
```

Comment:
```suggestion
    The data of the CV can be returned as a dataframe with axis 't', 'U', and 'I' (current) or 'j' (current density).
```
Should we really with ```U``` instead of ```E```?